.github/workflows/release.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: '10.14'
+          node-version: 12
```

Comment:
Should this be 12 or the latest 12.x.y as we do in Parse Server? And should we add multiple Node versions like we have in Parse Server?

Reply:
This change is only for the release and I am not sure how multiple versions would work. We are currently testing node 12 and 14 for tests ci: https://github.com/parse-community/node-apn/blob/master/.github/workflows/ci.yml#L12

I am not sure if we should specify a minor version. I believe that, specifying as 12, it will use the latest minor under 12, which looks good to me. What do you thing?

Reply:
Ah right, that looks good then.